src/Components/OATPropertyEditor/Internal/PropertyList/Internal/PropertyListItem/Internal/PropertyListItemChildHost/Internal/PropertyListItemMapChild/PropertyListItemMapChild.tsx

```diff
@@ -45,8 +72,12 @@ const PropertyListItemMapChild: React.FC<IPropertyListItemMapChildProps> = (
                 onCopy={undefined}
                 onRemove={undefined}
                 onReorderItem={undefined}
-                onUpdateName={undefined}
-                onUpdateSchema={undefined}
+                onUpdateName={(args) => {
+                    const keyCopy = deepCopy(item.mapKey);
```

Comment:
You use a very similar function in the following onUpdateName and onUpdateSchema, could we pull this out into a shared function?

Reply:
one updates the name property and one updates schema so I think this is about as consolidated as they can be.